src/app/test/page.tsx

```diff
@@ -1,43 +1,55 @@
 'use client'
 
-import { UUID } from 'crypto'
 import { useEffect, useState } from 'react'
-import { createUser, getUser } from '@/utils/userUtils'
+import { getAttendees, addAttendee } from '@/utils/attendeesUtils'
+import { UUID } from 'crypto'
+
+interface Attendee {
```

Comment:
can this be in the atteneesUtils for re-use?

Reply:
Moved Attendee interface to attendeeUtils.